canvas_modules/common-canvas/__tests__/_utils_/common-canvas-utils.js

```diff
@@ -68,7 +68,8 @@ export function createIntlCommonCanvas(
 				toolbarConfig={toolbarConfig}
 				notificationConfig={notificationConfig}
 				contextMenuConfig={contextMenuConfig}
-				showRightFlyout={false}
+				showRightFlyout={showRightFlyout}
+				showBottomPanel={}
```

Comment:
You are getting a build problem because of a lint error with this line. You should be passing the `showBottomPanel` parameter in here instead of `{}` otherwise there is no point in passing in that parameter to the function.